components/Footer.js

```diff
@@ -47,29 +47,29 @@ export default function Footer() {
         </nav>
         <div className="mt-8 flex justify-center">
           <a href="https://twitter.com/AsyncAPISpec" target="_blank" rel="noopener noreferrer" className="text-gray-400 hover:text-blue-500">
-            <span className="sr-only">Twitter</span>
+            <span className="sr-only">Follow AsyncAPI on Twitter</span>
             <IconTwitter className="h-6 w-6" />
           </a>
           <a href="https://github.com/asyncapi" target="_blank" rel="noopener noreferrer" className="ml-6 text-gray-400 hover:text-gray-500">
-            <span className="sr-only">GitHub</span>
+            <span className="sr-only">AsyncAPI on GitHub</span>
             <IconGithub className="h-6 w-6" />
           </a>
           <a href="https://linkedin.com/company/asyncapi" rel="noopener noreferrer" target="_blank" className="ml-6 text-gray-400 hover:text-blue-500">
-            <span className="sr-only">LinkedIn</span>
+            <span className="sr-only">AsyncAPI on LinkedIn</span>
             <IconLinkedIn className="h-6 w-6" />
           </a>
           <a href="https://youtube.com/asyncapi" target="_blank" rel="noopener noreferrer" className="ml-6 text-gray-400 hover:text-red-600">
-            <span className="sr-only">YouTube</span>
+            <span className="sr-only">Subscribe to AsyncAPI on YouTube</span>
             <IconYoutube className="h-6 w-6" />
           </a>
         </div>
         <div className="mt-8">
           <p className="text-center text-base leading-6 text-gray-400">
-            Made with <span className="font-mono text-pink-500">:love:</span> by the AsyncAPI Initiative.
+            Made with <span className="font-mono text-pink-500">❤️</span> by the AsyncAPI Initiative.
```

Comment:
This `:love:` is meant to be text only. I already confirmed it with Lukasz earlier. You probably have to undo the changes here.

Reply:
reverted

Reply:
@fmvilas I know you did it intentionally, but this is already a 3 time, 3rd PR from another person that wants to fix that because it feels like a bug. IMHO we should do a real ❤️ there

it is as ofter as PRs to fix docs because people think we mixed publish subscribe definition 😄